src/Tasks/AssemblyDependency/ReferenceTable.cs

```diff
@@ -407,6 +407,7 @@ internal void AddReference(AssemblyNameExtension assemblyName, Reference referen
                 }
             }
 
+            reference.FullPath = FileUtilities.NormalizePath(reference.FullPath);
```

Comment:
comment reasoning please -- "Saves effort and makes deduplication possible downstream" or something like that?

Also changewave it? I don't know why it would cause problems but . . .